import/WebViewPage.qml

```diff
@@ -0,0 +1,26 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+
+Page {
+    id: webViewPage
+
+    property var activeWebView
+
+    property int __sailfish_webviewpage
+
+    states: [
+        State {
+            name: "webViewActive"
+            when: activeWebView != null && activeWebView != undefined && activeWebView.visible && activeWebView.active
+            PropertyChanges {
+                target: pageStack
+                _noGrabbing: activeWebView.moving || activeWebView.pinching || activeWebView.dragging
+            }
+            PropertyChanges {
+                target: webViewPage
+                backNavigation: activeWebView.atXBeginning && !activeWebView.pinching
```

Comment:
Let's see how this behaves. We can iterate. I'd like to get this in asap so that we can iterate and things wouldn't be blocked.